src/Robo/Plugin/Traits/DatabaseDownloadTrait.php

```diff
@@ -56,19 +56,20 @@ public function databaseDownload(string $siteName = 'default')
         usort($objects, fn($a, $b) => $a->getLastModified()->getTimestamp() <=> $b->getLastModified()->getTimestamp());
         $latestDatabaseDump = array_pop($objects);
         $dbFilename = $latestDatabaseDump->getKey();
+        $localFilename = $this->sanitizeFileNameForWindows($dbFilename);
```

Comment:
What are your thoughts on naming this something like `$downloadFileName` to help reinforce how it's being used?

Reply:
Shamelessly playing the Non-Native-Speaker card here. (And yes, looking at it that's a much better name)